prime-router/build.gradle.kts

```diff
@@ -126,8 +128,6 @@ val compileKotlin: KotlinCompile by tasks
 val compileTestKotlin: KotlinCompile by tasks
 compileKotlin.kotlinOptions.jvmTarget = appJvmTarget
 compileKotlin.kotlinOptions.allWarningsAsErrors = true
-// if you set this to true, you will get a warning, which then gets treated as an error
-compileKotlin.kotlinOptions.useK2 = false
```

Comment:
This is deprecated now (and might not have been required).  If we did need to force the version there is a `language` version option.